apps/helpers/tests/md.py

```diff
@@ -0,0 +1,93 @@
+"""Helper to get Travis/Codecov badges in markdown table."""
```

Comment:
This file is not needed here, we are working with custom_api. 

Delete this file from frontend app.